common/web/input-processor/src/text/prediction/languageProcessor.ts

```diff
@@ -210,10 +210,15 @@ export default class LanguageProcessor extends EventEmitter<LanguageProcessorEve
       throw "Accepting suggestions requires a destination OutputTarget instance."
     }
 
+    if(!this.isConfigured) {
+      console.warn("Can't apply suggestions until model is fully loaded");
```

Comment:
```suggestion
      // If we're in this state, the suggestion is now outdated; the user must have swapped keyboard and model.
      console.warn("Could not apply suggestion; the corresponding model has been unloaded");
```

Reply:
Note that suggestions are only generated by fully-configured, loaded models.  To be unconfigured at this point means that the system is in a state where the suggestion's model has been recently unloaded.

For this to happen, there probably must be a backlog of events - the suggestion _was_ current when pressed, but the current state no longer supports it.  (Also consider the old TWo caps bug - where two OSK keys were tapped simultaneously before the first key's layer swap could take effect.)  Essentially, I believe this to be a variant on the Android rapid-typing bugs we were experiencing for gestures in 17.0-beta, which were due to the same underlying event-backlog concept.

Also note that JS does keep backlogged events in FIFO order.  If the suggestion's model is no longer available, that likely means someone tapped it after triggering a keyboard swap with an earlier backlogged event.  We're playing catchup, and the queued event order means that the suggestion wouldn't have been available if not for the event-backlog.

Unsurprisingly, the Sentry logs for the base issue are coming from Android devices.  (Exclusively, actually.)